include/wx/webview.h

```diff
@@ -78,6 +78,20 @@ enum wxWebViewFindFlags
     wxWEBVIEW_FIND_DEFAULT =          0
 };
 
+enum wxWebViewIEEmulationLevel
```

Comment:
I think we should only define this and the methods in webview_ie.h as they aren't ever going to be used elsewhere.